include/matx/generators/random.h

```diff
@@ -178,38 +177,17 @@ template <typename T> class randomGenerator_t {
   inline randomGenerator_t(index_t total_threads, uint64_t seed)
       : total_threads_(total_threads)
   {
-#ifdef __CUDACC__      
+#ifdef __CUDACC__
     matxAlloc((void **)&states_,
               total_threads_ * sizeof(curandStatePhilox4_32_10_t),
               MATX_DEVICE_MEMORY);
 
     int threads = 128;
     int blocks = static_cast<int>((total_threads_ + threads - 1) / threads);
     curand_setup_kernel<<<blocks, threads>>>(states_, seed, total_threads);
-#endif    
+#endif
   };
 
-  /**
-   * Get a tensor view of the random numbers
-   *
-   * @param shape
-   *   Dimensions of the view in the form of an tensorShape_t
-   * @param dist
-   *   Distribution to use
-   * @param alpha
-   *   Alpha value
-   * @param beta
-   *   Beta value
-   * @returns
-   *   A randomTensorView_t with given parameters
-   */
-  template <int RANK>
-  inline auto GetTensorView(const tensorShape_t<RANK> shape,
-                            Distribution_t dist, T alpha = 1, T beta = 0)
-  {
-    return randomTensorView_t<T, RANK>(shape, states_, dist, alpha, beta);
-  }
-
   /**
    * Get a tensor view of the random numbers
```

Comment:
This looks like it should be `shape` still

Reply:
updated this comment and added a function header to the overloaded one below